app/views/analyses/new.html.erb

```diff
@@ -5,21 +5,20 @@
     <h1>Select Geography</h1>
     <section id="slide-1" class="l-slide -theme-1 js--slide -is-current">
       <div class="c-slide-content">
-        <div class="c-map-group">
-          <div id="mapCountries" class="c-map"></div>
-          <!-- <div id="mapRegions" class="c-map"></div> -->
-        </div>
+        <div id="mapCountries" class="c-map"></div>
         <%= label_tag :country, "Select your country", class: "text -questions country-text" %>
         <%= label_tag :region, "Select your region", class: "text -questions is-hidden region-text" %>
         <p class="text -description country-text">Start typing or select your country in the map</p>
         <p class="text -description region-text is-hidden">Start typing or select your region in the map</p>
         <span class="c-input-field">
           <%= select_tag :country,
-            options_from_collection_for_select(@geo_locations, :country, :country),
-            include_blank: true, class: '-input-big is-required' %>
+          options_from_collection_for_select(@geo_locations, :country, :country),
+          include_blank: true, class: '-input-big is-required'%>
         </span>
         <span class="c-input-field is-hidden">
-          <%= f.select :geo_location_id, [], {}, class: '-input-big' %>
+          <%= select_tag :country,
```

Comment:
With this change now when the country is selected it shows the countries inside the second selector as well.